documentation/book/proc-deploying-kafka-connect-kubernetes.adoc

```diff
@@ -3,20 +3,24 @@
 // assembly-kafka-connect.adoc
 
 [id='deploying-kafka-connect-kubernetes-{context}']
-= Deploying Kafka Connect to {KubernetesName}
+= Deploying Kafka Connect to your {KubernetesName} cluster
```

Comment:
Is there any reason why do we have two procedures ... one for Kube and one for OpenShift? Most other procedures are one procedure for both.

Reply:
I'm not sure why this is the case here.